app/src/main/java/org/literacyapp/authentication/StudentImageCollectionActivity.java

```diff
@@ -49,6 +55,7 @@
     private LiteracyApplication literacyApplication;
     private List<Mat> studentImages;
     private AnimalOverlayHelper animalOverlayHelper;
+    private AnimalOverlay animalOverlay;
 
     // Image collection parameters
     private static final boolean DIAGNOSE_MODE = true;
```

Comment:
I think so, yes. Or maybe false by default and if we are testing, we can set it to true on our computers?